infrastructure/nginx/nginx.conf

```diff
@@ -27,5 +27,7 @@ http {
           proxy_redirect off;
           proxy_pass http://django:8080;
         }
+
+        add_header X-Frame-Options sameorigin;
```

Comment:
This is only used for the local dev environment, has not effect to Google App Engine.